<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Journal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="4">
     <widget class="QPushButton" name="btn_todo">
      <property name="toolTip">
       <string>Press &quot;T&quot;</string>
      </property>
      <property name="text">
       <string>Todo</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/icons.qrc">
        <normaloff>:/Todo.png</normaloff>:/Todo.png</iconset>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="lbl_profit">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Profit: 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="lbl_buy">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Buy: 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="lbl_date">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Date: DD/MM/YYYY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="lbl_sell">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Sell: 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="btn_delete_record">
      <property name="toolTip">
       <string>Press &quot;D&quot;</string>
      </property>
      <property name="text">
       <string>Delete Record</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/icons.qrc">
        <normaloff>:/Delete_record.png</normaloff>:/Delete_record.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_suggestions">
      <property name="toolTip">
       <string>Press &quot;U&quot;</string>
      </property>
      <property name="text">
       <string>Suggestions</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/icons.qrc">
        <normaloff>:/Suggestions.png</normaloff>:/Suggestions.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_statistics">
      <property name="toolTip">
       <string>Press &quot;S&quot;</string>
      </property>
      <property name="text">
       <string>Statistics</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/icons.qrc">
        <normaloff>:/Statistics.png</normaloff>:/Statistics.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btn_about_us">
      <property name="toolTip">
       <string>Press &quot;B&quot;</string>
      </property>
      <property name="text">
       <string>About us</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/icons.qrc">
        <normaloff>:/About_us.png</normaloff>:/About_us.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn_contact_us">
      <property name="toolTip">
       <string>Press &quot;C&quot;</string>
      </property>
      <property name="text">
       <string>Contact us</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/icons.qrc">
        <normaloff>:/Contact_us.png</normaloff>:/Contact_us.png</iconset>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="btn_add_record">
      <property name="toolTip">
       <string>Press &quot;A&quot;</string>
      </property>
      <property name="text">
       <string>Add record</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/icons.qrc">
        <normaloff>:/Add_record.png</normaloff>:/Add_record.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="6" colspan="4">
     <widget class="QTableWidget" name="tbl_widget_existing_records"/>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="btn_refresh">
      <property name="toolTip">
       <string>Press &quot;R&quot;</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/icons.qrc">
        <normaloff>:/Refresh.png</normaloff>:/Refresh.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
